ocaml/xapi/dbsync_slave.ml

```diff
@@ -140,17 +140,15 @@ let record_host_memory_properties ~__context =
 
   let metrics = Db.Host.get_metrics ~__context ~self in
   Db.Host_metrics.set_memory_total ~__context ~self:metrics ~value:total_memory_bytes;
-  let boot_memory_file = Xapi_globs.initial_host_free_memory_file in
-  let boot_memory_string =
+  let boot_memory_bytes =
     try
-      Some (Unixext.string_of_file boot_memory_file)
+      Some (Memory_client.Client.get_host_initial_free_memory "dbsync")
```

Comment:
Instead of `"dbsync"`, we should pass in a `dbg` value from `let dbg = Context.string_of_task __context in` (as above in `record_host_memory_properties`).